examples/EJS/nodejs/template.ejs

```diff
@@ -0,0 +1,4 @@
+<h1>Primes from <%= a %> to <%= b %></h1>
+<ul>
+  <% for(var i = 0; i < x.length; ++i) { %> <li><%= x[i] %></li> <% } %>
```

Comment:
You don't need whitespace between `%>` and `<`. Whitespace is sometimes meaningful in HTML (if the blocks are inline, the space gets a nonzero width).